tests/integration_tests/functional/test_api.py

```diff
@@ -931,7 +926,12 @@ def _drive_patch(test_microvm):
             "path_on_host": None,
             "rate_limiter": None,
             "io_engine": None,
-            "socket": "/vub.socket",
+            "socket": str(
+                Path("/")
+                / os.path.basename(
+                    test_microvm.disks_vhost_user["scratch_vub"].socket_path
```

Comment:
`socket_path` is a `Path`, no? It can then be `Path("/") / test_microvm.disks_vhost_user["scratch_vub"].socket_path.name`

Reply:
the reason I use `basename` is because the `socket_path` is host-side view, not Firecracker/jailer view.

Reply:
But `os.path.basename` and `Path.name` should return the same thing no? https://docs.python.org/3/library/pathlib.html#correspondence-to-tools-in-the-os-module

Reply:
I misunderstood you, done!